webpack.config.js

```diff
@@ -63,7 +63,7 @@ module.exports = [
    * User adds two script tags to their html page:
    *
    *   <script src="//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.js"></script>
```

Comment:
this is just pseudocode simulating what a user might do; totally possible for them to pull in d3 through unpkg or anywhere else